frontend/src/components/Canvas/canvas.scss

```diff
@@ -0,0 +1,7 @@
+@use '../Toolbox/toolbox.scss' as toolbox;
+
```

Comment:
When you name a component `Canvas` you should name the scss class the same, with capital letter